exp-models/addon/utils/eligibility.js

```diff
@@ -0,0 +1,129 @@
+// h/t @aaxelb
+
+import Ember from 'ember';
+import moment from 'moment';
+
+let {$} = Ember;
+
+var AND = (a, b) => {
+    return function() {
+        return a(...arguments) && b(...arguments);
+    };
+};
+var OR = (a, b) => {
+    return function() {
+        return a(...arguments) || b(...arguments);
+    };
+};
+
+/**
+ * @return {integer}: number of days represented by amount/unit pair, e.g. 3 'weeks' -> 21
+ **/
+function parseDate(amount, unit='days') {
+    var singular = unit.replace(/s$/, '');
+    return moment.duration(parseFloat(amount), `${singular}s`).asDays();
+}
+
+/** Compile an eligibiliy string into a function: (participant) -> boolean
+ *
```

Comment:
Ah, example was added before this PR. Thanks!

Since I don't see an easy way to validate this with a simple regex, we'll definitely want to keep this documentation when it comes time for users to define experiments.